staging/src/k8s.io/code-generator/cmd/validation-gen/main.go

```diff
@@ -88,6 +89,8 @@ func (args *Args) AddFlags(fs *pflag.FlagSet) {
 		"the path to a file containing boilerplate header text; the string \"YEAR\" will be replaced with the current 4-digit year")
 	fs.BoolVar(&args.PrintDocs, "docs", false,
 		"print documentation for supported declarative validations, and then exit")
+	fs.BoolVar(&args.LintOnly, "lint-only", false,
```

Comment:
How about just `--lint` - I suspect this pattern will be copied :)

Reply:
Updated.